en/tutorials/linux_sbus.md

```diff
@@ -23,13 +26,3 @@ GND ->S.bus gnd
 
 ## Circuit diagram
 ![](http://www.playuav.com/uploads/article/20160310/56cf0f65bb1f7437c1618041a30dc308.png)
-
-## Launch the driver
```

Comment:
Why did you remove this? I just asked to show where to add the driver start on the config files.